### PR TITLE
Update ticket status: Student Ticket

### PR DESCRIPTION
**Pull request by vipulgupta2048**
Since, Student tickets are sold out. 

Signed-off-by: Vipul Gupta (@vipulgupta2048) <vipulgupta2048@gmail.com>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -242,7 +242,7 @@ <h3>Armin Ronacher</h3>
                                       <div class="col-xs-3">
                                         <a href="http://lucumr.pocoo.org" title="Visit My Website"><i class="fas fa-globe"></i></a>
                                       </div>
-                                    </div>  
+                                    </div>
```
Comment (realslimshanky) [inline index.html]: Ideally, this should not be part of the diff.

Comment (realslimshanky) [inline index.html]: Ideally, this should not be part of the diff.

File: index.html [hunk 2]
```diff
@@ -269,7 +269,7 @@ <h3>Carol Willing</h3>
                                         <div class="col-xs-3">
                                           <a href="http://willingconsulting.com" title="Visit My Website"><i class="fas fa-globe"></i></a>
                                         </div>
-                                      </div>  
+                                      </div>
```
Comment (realslimshanky) [inline index.html]: Ideally, this should not be part of the diff.

File: index.html [hunk 3]
```diff
@@ -296,7 +296,7 @@ <h3>Travis E. Oliphant</h3>
                                       <div class="col-xs-3">
                                         <a href="http://technicaldiscovery.blogspot.com/" title="Visit My Blog"><i class="fas fa-globe"></i></a>
                                       </div>
-                                    </div>  
+                                    </div>
```
Comment (realslimshanky) [inline index.html]: Ideally, this should not be part of the diff.

File: index.html [hunk 4]
```diff
@@ -863,13 +863,13 @@ <h3 class="time">Student Ticket</h3>
                                     <b>Note:</b> Need to produce valid student ID card to collect their entry passes during the conference
                               </div>
                               <div class="col-md-2 col-xs-12 text-center ticket-column">
-                                <span class="center-align"> 
+                                <span class="center-align">
```
Comment (realslimshanky) [inline index.html]: Ideally, this should not be part of the diff.

**Comment by vipulgupta2048** [comment]
@realslimshanky Thanks for the review. My editor just removed whitespace of it. I will make the changes as requested.

**Comment by inovizz** [comment]
One comment here - we are planning to increase student tickets number. So can you keep this on hold for a few days.

**Comment by vipulgupta2048** [comment]
That's great news @inovizz, Putting it on hold.

**Comment by vipulgupta2048** [comment]
@ananyo2012  I don't understand what I should update on the website, exactly. I mean the tickets were being shown before and now they are open as well. When they close this time, then we can just push this request.

**Comment by ananyo2012** [comment]
We need to update the total no of tickets. Just for the updated info. Same
in the blog. It will be helpful later in next PyCons when we refer back.

**Comment by vipulgupta2048** [comment]
... and how much would that be. Earlier it was 300 student tickets.

**Comment by ananyo2012** [comment]
@vipulgupta2048 Student tickets are sold out. We can resume work on this now.

File: index.html [hunk 5]
```diff
@@ -255,20 +255,20 @@ <h5 class="text-center">Creator of Flask</h5>
...
-                                <div class="vcenter-wrapper">
-                                  <div class="centered">
-                                    <h3>Carol Willing</h3>
-                                    <div class="row text-center social-links">
-                                      <div class="col-xs-3 col-xs-offset-3">
-                                        <a href="https://twitter.com/WillingCarol" title="@WillingCarol"><i class="fab fa-twitter"></i></a>
-                                      </div>
-                                      <div class="col-xs-3">
-                                        <a href="http://willingconsulting.com" title="Visit My Website"><i class="fas fa-globe"></i></a>
+                              <div class="bordered-image">
```
Comment (ananyo2012) [inline index.html]: Seems like there is extra spaces added for indentation here. Can you fix this?

**Comment by vipulgupta2048** [comment]
The changes have been made, merging it in.

**Pull request merged**